packages/vue/src/Mount.vue

```diff
@@ -6,20 +6,22 @@
 import type {
   BasePath,
   ListenFunc,
+  LogAnyError,
   MountFunc,
   NavigateFunc,
   RemoveListener,
   Runtime,
   UnmountFunc,
 } from "@leanjs/core";
-import { onBeforeUnmount, ref, watchEffect } from "vue";
+import { onBeforeUnmount, onMounted, ref } from "vue";
 
 export interface MountProps extends BasePath {
   mount: MountFunc;
   runtime: Runtime;
   basename?: string;
   navigate?: NavigateFunc;
   listen?: ListenFunc;
+  setError: LogAnyError;
```

Comment:
should we use `setError` instead of throwing the error when calling `mount`, `onError` argument, on line 48?